engine/runtime/src/main/java/org/enso/interpreter/runtime/error/WithWarnings.java

```diff
@@ -250,6 +286,19 @@ boolean hasSpecialDispatch() {
     return true;
   }
 
+  /*
```

Comment:
Interesting. Do I understand correctly by this commented snippet, that you have tried to `@ExportLibrary(InteropLibrary.class)` and export these two messages and you have somehow failed? Because you cannot export both `InteropLibrary` and `ReflectionLibrary` in the same type?

Reply:
Yes, that's correct. If I export `InteropLibrary`, then I am supposed to delegate all messages manually. Many of the `InteropLibrary` messages aren't implemented in the `WithWarnings` at all - right now they are handled by `ReflectionLibrary`, but when I add `@ExportLibrary(InteropLibrary.class)` they are sent to default implementation instead.

Reply:
Rewritten in [3fe42f0](https://github.com/enso-org/enso/pull/9840/commits/3fe42f0e1ef74c67b36c759bdb4faf1da32d23a3) to `delegateTo`.